extension/main.js

```diff
@@ -13,6 +13,15 @@
 		chrome.browserAction.setTitle({
 			title: title
 		});
+
+		if (badge && window.GitHubNotify.settings.get('showDesktopNotif')) {
+			chrome.notifications.create('github-notifier', {
+				title: 'GitHub',
+				iconUrl: 'icon-128.png',
+				type: 'basic',
+				message: 'You have new unread notifications'
```

Comment:
A <kbd>View</kbd> button would be redundant since we could ideally make clicking on the notification itself open the issue in question.

That said, I fully agree with @sindresorhus in that it would be more useful to show the actual notifications themselves. I'd go with one notification per issue.